resources/compass/values.yaml

```diff
@@ -143,18 +134,24 @@ global:
       host: adapter-gateway
     mtls:
       host: adapter-gateway-mtls
-  
+
   rewriteFilters:
     workloadLabel: oathkeeper
     tokenDataHeader: "Connector-Token"
     certificateDataHeader: "Certificate-Data"
 
+  ingress:
+    domainName: kyma.local
```

Comment:
I wonder if we should keep it, as these values are not specified for all Kyma components. We have override for local installation for our `kyma.local` domain. By removing them from here we enforce users to specify their own configuration for the domain.

Reply:
Will remove the whole `ingress: ` block